src/Container/AutowireFactory.php

```diff
@@ -61,9 +65,11 @@ public function create(ContainerInterface $container, string $requestedName, ?ar
 
     /**
      * Make invokable and implement the laminas-service factory pattern
+     *
+     * @return bool
      */
-    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
+    public function __invoke(ContainerInterface $container, string $requestedName, ?array $options = null)
```

Comment:
Please revert his change as it introduces BC break.

Reply:
I removed the string Typehint. The ? before array was added by cs-fix.